iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers.java

```diff
@@ -57,11 +57,10 @@ private void replaceThreeWindingsTransformerBy3TwoWindingsTransformer(ThreeWindi
         regulatedTerminalControllers.replaceRegulatedTerminal(t3w.getLeg2().getTerminal(), t2wLeg2.getTerminal1());
         regulatedTerminalControllers.replaceRegulatedTerminal(t3w.getLeg3().getTerminal(), t2wLeg3.getTerminal1());
 
-        // t2wLeg1, t2wLeg, and t2wLeg3 are not considered in regulatedTerminalControllers (created in the model later)
-        Map<Terminal, Terminal> regulatedTerminalMapping = getRegulatedTerminalMapping(t3w, t2wLeg1, t2wLeg2, t2wLeg3);
-        replaceRegulatedTerminal(t2wLeg1, regulatedTerminalMapping);
-        replaceRegulatedTerminal(t2wLeg2, regulatedTerminalMapping);
-        replaceRegulatedTerminal(t2wLeg3, regulatedTerminalMapping);
+        // t2wLeg1, t2wLeg, and t2wLeg3 are not considered in regulatedTerminalControllers (created later in the model)
+        replaceRegulatedTerminal(t2wLeg1, t3w, t2wLeg1, t2wLeg2, t2wLeg3);
```

Comment:
Can you introduce a new record containing "t2wLeg1, t2wLeg2, t2wLeg3"? It will improve readability and may prevent typo-bugs. 
(There are 8 methods taking this sequence of the 3 two winding transformers.)

Reply:
Done